source/operations/checklists/hardware.rst

```diff
@@ -513,3 +461,63 @@ Document the performance numbers for each server in your deployment.
      - Number of threads (:math:`numberOfDrives * 16`)
    * - ``-F <>``  
      - list of files (the above command tests with 16 files per drive)  
+
+Recommended tools for MinIO subscriptions
+-----------------------------------------
+
+.. important::
+
+   The tools noted in this section **require** a MinIO subscription.
+   For more information, see the `MinIO pricing page <https://min.io/pricing?jmp=docs>`
```

Comment:
```suggestion
.. important::

   The tools noted in this section **require** a MinIO subscription.
   MinIO strongly recommends all production deployments use the `AIStor Object Store <https://min.io/docs/aistor-custom/object-store/>`  with their SUBNET license.
   For more information, see the `MinIO pricing page <https://min.io/pricing?jmp=docs>`
```

Reply:
🤔 re: Ravind's suggestion

_the_ AIStor Object Store
or just
AIStor Object Store

Reply:
Dropping the article and calling it the MinIO AIStor pricing page.